recipes/debarcer/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set version = "2.1.1" %}
+
+package:
+  name: debarcer
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/d/debarcer/debarcer-{{ version }}.tar.gz
+  sha256: 976d86e7460555cf6e86985969727ca0daf4c5e28b647a66ef4a6d3c0dc99a40
+
+build:
+  number: 0
+  entry_points:
+    - debarcer = debarcer.debarcer:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    # importlib_metadata is needed for py<38 only, but conda does not support conditional
```

Comment:
mh, what is this? Conda does support conditional dependencies.

Reply:
Good point. That was a remaining from copy&pasting a template recipe.